etc/config/c++.amazon.properties

```diff
@@ -1152,6 +1152,7 @@ libs.nlohmann_json.versions.211.version=2.1.1
 libs.nlohmann_json.versions.211.path=/opt/compiler-explorer/libs/nlohmann_json/v2.1.1/src
 libs.jsoncpp.name=JsonCpp
 libs.jsoncpp.description=JsonCpp is a C++ library that allows manipulating JSON values, including serialization and deserialization to and from strings
+libs.jsoncpp.examples=o1W1jofna:rfPajea5G
```

Comment:
I think it's the only thing needed, by looking at other libraries, but I could have missed something. Let me know if an extra step is required

Reply:
Yup, this is all